Sources/TripKitUI/views/trip overview/TKUIAttributionView.swift

```diff
@@ -58,17 +58,14 @@ public class TKUIAttributionView: UIView {
   fileprivate func didInit() {
     backgroundColor = .tkBackground
     
-    let textView = UITextView()
-    textView.font = TKStyleManager.customFont(forTextStyle: .footnote)
-    textView.backgroundColor = .clear
-    textView.textColor = .tkLabelSecondary
-    textView.isEditable = false
-    textView.isScrollEnabled = false
-    textView.isPagingEnabled = false
-    textView.dataDetectorTypes = [.phoneNumber, .link, .address]
```

Comment:
what was substituted for this for UILabel switch?

Reply:
We handle tapping on the attribution separately to bring up the full list and not use the data detectors anymore. Text views are harder to handle for properly sizing, so a label is simpler and preferred.